src/client/styles/components/_form-component.scss

```diff
@@ -0,0 +1,13 @@
+.message {
+    border-top: 2px solid #ffb81c;
+    background-color: #fff8e8;
+    color: #825900;
+    padding: 32px;
+}
+.address-message {
+  clear:both;
+}
+.address-message-elem {
```

Comment:
Might be worth looking at some of these class names. NHS Frontend and other repos within NHSUK have been following the [BEM](http://getbem.com/introduction/) methodology for naming classes which was also set out in the original organisation [styleguide](https://github.com/nhsuk/styleguides/blob/master/css.md).